integritybackend/actions.py

```diff
@@ -53,12 +54,13 @@ def archive(self, asset_fullpath: str, org_config: dict, collection_id: str):
 
         # Pull things out of the config
         collection_config = next(
-            (c for c in org_config["collections"] if c.get("id") == collection_id), None
+            (c for c in org_config["collections"] if org_config["collections"][c]['conf']['id'] == collection_id), None
         )
         if collection_config is None:
             raise Exception(
                 f"No collection in {org_config['id']} config with ID {collection_id}"
             )
+        collection_config=org_config["collections"][collection_config]['conf']
```

Comment:
collection_config is assigned to c which is a string, not a object.

Reply:
`collection_config` is actually the full collection config, not the key.

Reply:
Debug code:
```
        # Pull things out of the config
        collection_config = next(
            (c for c in org_config["collections"] if org_config["collections"][c]['conf']['id'] == collection_id), None
        )
        print("$$$$$$$$$$$$$$")
        print(collection_config)
        print("$$$$$$$$$$$$$$")
```
result
```
Apr  4 04:04:50 integrity pipenv[27475]: $$$$$$$$$$$$$$
Apr  4 04:04:50 integrity pipenv[27475]: test-bot-archive
Apr  4 04:04:50 integrity pipenv[27475]: $$$$$$$$$$$$$$
```